File: cuatro_numeros.cpp
//Kleber Jesus Montes Compañ
//Ing Animacion digital
//00623774
#include <iostream>
#include <cstdlib>
using namespace std;

int main () {

int num1, num2, num3, num4, May;

cout << "Digite numero 1: " << endl;
cin >> num1;
cout << "Digite numero 2: " << endl;
cin >> num2;
cout << "Digite numero 3: " << endl;
cin >> num3;
cout << "Digite numero 4: " << endl;
cin >> num4;

May = num1;
if (num2 > May) May = num2;
if (num3 > May) May = num3;
if (num4 > May) May = num4;

cout << "El numero mayor es el: " << May << endl;

system("pause");

return 0;

}

File: cuatro_numeros.py
#Kleber Jesus Montes Compañ
#Ingenieria en Animacion Digital
#ID: 00623774 """
print ("Digite numero 1: ")
num1 = int(input())
print ("Digite numero 2: ")
num2 = int(input())
print ("Digite numero 3: ")
num3 = int(input())
print ("Digite numero 4: ")
num4 = int(input())

if (num1 > num2 and num1 > num3 and num1 > num4):
    May = num1
elif (num2 > num1 and num2 > num3 and num2 > num4):
    May = num2
elif (num3 > num1 and num3 > num2 and num3 > num4):
    May = num3
else: May = num4

print ("El numero mayor es el: ", May)